src/lib/constant_mappings.cpp

```diff
@@ -52,6 +52,7 @@ const boost::bimap<VectorCompressionType, std::string> vector_compression_type_t
     make_bimap<VectorCompressionType, std::string>({
         {VectorCompressionType::FixedSizeByteAligned, "Fixed-size byte-aligned"},
         {VectorCompressionType::SimdBp128, "SIMD-BP128"},
+        {VectorCompressionType::FixedSizeBitAligned, "Fixed-size bit-aligned"},
```

Comment:
just a suggestion, it nicely describes the concept that the size is fixed. The abbreviation _FSBA_ then obviously doesn't work anymore, we could call it _FSByteA_ and _FSBitA_ :D

Reply:
Wondering if `BitAlignedFixedSize` and `ByteAlignedFixedSize` makes it easier to digest.

Reply:
Sounds better and I would be fine with it. Would you rename FSBA in this PR or should I do this in a separate followup?

Reply:
@Bouncner

Reply:
I’d say in this PR, but let’s wait until we are certain that we will merge.

Reply:
So let's move that discussion here. First, @mrks  rightly pointed out that "bit-aligned" does not really make sense.
Second, we would like to ensure that Hyrise developers can easily differentiate the two compressions (even though they are quite similar) when reading the code. Third, the current "byte-aligned" compression is not really byte-aligned as we do not use a 24bit width but rather three integer types `uint(8/16/32_t`.

Current proposals:
- `BitCompression`
- `FixedWidthInteger`

Reply:
If we go this route, I think `Bitpacking` is the term that other authors often use to refer to the technique. Example: https://lemire.me/blog/2012/03/06/how-fast-is-bit-packing/

Reply:
so I'd call it "bitpacking" instead of "bitcompression" because both of those techniques are vector _compression_ schemes in the hyrise language

Reply:
`BitPacking` is fine with me. I've been googling a few compression papers but haven't found a term for "byte-aligned" stuff.

So any objections against `BitPacking` and `FixedWidthInteger`, @mrks, @mweisgut, and @Bensk1 ?

Reply:
i'll do it then